src/Components/ActivitiesCarousel/ActivitiesCarousel.css

```diff
@@ -0,0 +1,56 @@
+.ant-carousel .slick-prev,
+.ant-carousel .slick-next {
+  color: unset;
+  font-size: unset;
+  background: rgb(0, 0, 0, 0.4);
+  color: white;
+  height: 20px;
+  width: 50px;
+}
+.ant-carousel .slick-prev:hover,
+.ant-carousel .slick-next:hover,
+.ant-carousel .slick-prev:focus,
+.ant-carousel .slick-next:focus {
+  color: unset;
+}
+.slick-list {
+  width: 400px;
+}
+.ant-carousel {
+  width: 400px;
```

Comment:
you don't need this, lets make the size of the carousel dynamic so the wrapper element can specify the width and hight for their needs
like this for example
``` jsx
  <div style={{ height: '300px', width: '500px' }}>
        <ActivitiesCarousel actPicArray={actPicArray} />
   </div>
```